packages/lodestar-cli/src/commands/dev/command.ts

```diff
@@ -1,7 +1,7 @@
 /**
  * @module cli/commands
  */
-
+process.setMaxListeners(15);
```

Comment:
why?

Reply:
to remove that warning about max listeners. It's because we have 11 libraries that are subscribed on "unhandledRejection" event in process